lib/DDG/Spice/Aqi.pm

```diff
@@ -20,9 +20,31 @@ spice to => 'http://www.airnowapi.org/aq/observation/zipCode/current/?format=app
 spice wrap_jsonp_callback => 1;
 
 handle remainder => sub {
-     my $query = shift;
-     return unless $query =~ qr/^\d{5}$/; #check if the entire remainder string is a 5 digits number;
-     return $query; # we know it exists, it's a number and it has 5 digits;
+    my $query = shift;
+
+    if($query eq 'local') {
```

Comment:
@davidofferman now that I think about this a little more, I think it's fair to trigger this when there is no remainder as long as the user is in the US.

That way queries like "air quality" would show this, similar to how "weather" shows the local weather.